addons/payment_xendit/models/payment_transaction.py

```diff
@@ -8,7 +8,9 @@
 from odoo import _, models
 from odoo.exceptions import ValidationError
 
+from odoo.addons.payment import utils as payment_utils
 from odoo.addons.payment_xendit import const
+from odoo.tools import float_round
```

Comment:
Base Odoo imports should be in the same block: move odoo.toold after odoo.exceptions above.